integration/client-cli/admin-cli/src/main/java/org/keycloak/client/cli/common/BaseConfigCredentialsCmd.java

```diff
@@ -115,6 +115,7 @@ public void process() {
         } else if (keystore != null || secret != null || clientSet) {
             grantTypeForAuthentication = OAuth2Constants.CLIENT_CREDENTIALS;
             printErr("Logging into " + server + " as " + "service-account-" + clientId + " of realm " + realm);
+            printErr("WARN service-account-" + clientId + " is a temporary account. To harden security, create a permanent account and delete the temporary one.");
```

Comment:
I don't think this has to be a temporary account at this point.

Reply:
@shawkins We're telling to the user to consider the accounts temporary. We just don't expire them automatically at this point.

Reply:
@vmuzikar what I mean is that this warning will say that all clients are temporary - even those that are not.

Reply:
Ah, right. Indeed, that is not correct. :) But as I mentioned, I believe this warn belongs somewhere else anyway.